src/classes/STG_InstallScript.cls

```diff
@@ -250,4 +252,20 @@ global without sharing class STG_InstallScript implements InstallHandler {
         }
         return null;
     }
+
+    /*******************************************************************************************************
+    * @description Disables Preferred Email handling when upgrading from 1.30
+    * @return void
+    */
+    global static void disablePreferredEmailEnforcement(Version previousVersion) {
```

Comment:
Yes, because we are actually setting it to true here, for existing Orgs upgrading. Remember disabled == true for the setting. I know that seems backwards, but it kinda makes sense for me since all new Orgs will have it unchecked.